integration/css-modules-test.ts

```diff
@@ -20,6 +20,7 @@ test.describe("CSS Modules", () => {
   test.beforeAll(async () => {
     fixture = await createFixture({
       config: {
+        cssBundle: true,
```

Comment:
can we name it `unstable_cssBundle` and type it as just `false`? Make it a hard opt-out, as it needs to be ran if the package is included anyways.